code/game/area/areas.dm

```diff
@@ -102,6 +102,8 @@
 	var/list/air_vent_info = list()
 	var/list/air_scrub_info = list()
 
+	///How many people have eaten in this area? If it's above 40, the area is considered a nice place to eat. // todo: make this a scaling bonus deal instead of a threshold after the mood refactor
```

Comment:
as far as I can tell this isn't 40 anymore, which is why I wouldn't include hard numbers in documentation. Say "if it's above BREAK_ROOM_DESIGNATION"

Reply:
yeah sorry forgot to change that